### ISSUE TITLE
[Bug] `Map` and `Gamemode` variables never update

### ISSUE THREAD

**Issue by No-Half-Measures** [MrOkiDoki/BattleBit-Community-Server-API#62]
**Describe the bug**
The `Map` and `Gamemode` variables part of the `GameServer` class only store the launched map and gamemode and never change past that even when both have changed

**To Reproduce**
Add to the server class:
```
public override async Task OnRoundStarted()
{
    await Console.Out.WriteLineAsync($"Round Started - Map:{Map}, Gamemode:{Gamemode}");
}
```
Launch the server and API.
Join the server and force the round to start.
End the round and vote on a different map and gamemode to what started.
Force the new round to start and look at the output from `OnRoundStarted` in the API console.

**Expected behaviour**
The only out put will be the map and gamemode set in the server launch args.

**Screenshots**
![image](https://github.com/MrOkiDoki/BattleBit-Community-Server-API/assets/916733/0ce93a4e-3ccf-4b72-be74-59a04af51c02)

**Comment by RainOrigami** [comment]
I believe `CurrentPlayerCount` is affected too based on mentions from orell12

**Comment by mevenG** [comment]
Same for `SessionID`

**Comment by MapleEve** [comment]
It's been fixed in 2.0.9
